test/EstateRegistry.js

```diff
@@ -250,6 +250,11 @@ contract('EstateRegistry', accounts => {
         _data: metadata
       })
     })
+
+    it('fails if somebody else tries to steal land', async function() {
```

Comment:
I think we could add a test also checking if creator or operator can only create an Estate

Reply:
👍